core/shared/src/test/scala/com/monovore/decline/ArgumentSpec.scala

```diff
@@ -60,6 +63,53 @@ class ArgumentSpec extends ArgumentSuite {
   checkArgument[BigInt]("BigInt")
   checkArgument[UUID]("UUID")
 
+  val onlyValidArbitraryDuration = Arbitrary(
```

Comment:
The default `Arbitrary[Duration]` scalacheck property generator also generates `Duration.Undefined` which we don't care (there is no point in generating decline working with undefined `Duration`).